tests/Feature/Controllers/FeaturedControllerTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Tests\Feature\Controllers;
+
+use App\Contracts\AlbumRepositoryInterface;
+use App\Contracts\ArtistRepositoryInterface;
+use App\Contracts\FeaturedRepositoryInterface;
+use CountriesSeeder;
+
+class FeaturedControllerTest extends ControllerTestCase
+{
+    /**
+     * @var FeaturedRepositoryInterface $featured
+     */
+    protected $featured;
+
+    /**
+     * @var ArtistRepositoryInterface $artist
+     */
+    protected $artist;
+
+    /**
+     * @var AlbumRepositoryInterface $album
+     */
+    protected $album;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed(CountriesSeeder::class);
+
+        $this->featured = resolve(FeaturedRepositoryInterface::class);
+
+        $this->artist = resolve(ArtistRepositoryInterface::class);
+
+        $this->album = resolve(AlbumRepositoryInterface::class);
+    }
+
+    public function testAllReturnsOkStatusAndExpectedJsonStructure()
+    {
+        $artist = $this->createArtist();
+
+        $this->makeAlbum([
+            'artist_id' => $artist->id,
+            'is_active' => true,
```

Comment:
Didn't you remove the `is_active` from the structure ?

Reply:
I remove it from the `Featured` model structure, not from the `Album`, which is what we're looking at here.